### ISSUE TITLE
Control-Flow Breaking

### ISSUE THREAD

**Issue by Hcount** [open-obfuscator/o-mvll#67]
Thanks for providing such an excellent tool!

Is there any plan to support iOS for Control-Flow Breaking.

Cheers!
-Zack

**Comment by marcobrador** [comment]
Yes, work on improving and extending obfuscation passes is planned for H1-2025.

So far work on stabilising a few of them and updating to meet latest Xcode versions has taken much of our bandwidth, together with upcoming support for ARM32.

We expect this to change now and be able to focus on pure obfuscation rather than "infrastructure" topics.

**Comment by marcobrador** [comment]
Dear @Hcount ,
We've made some fixes to BreakCFG in the [latest release](https://github.com/open-obfuscator/o-mvll/releases/tag/1.3.0). Would you mind trying it out?
Thanks,
Marc

**Comment by meiyingda1** [comment]
Thanks for providing such an excellent tool!
def break_control_flow(self, mod: omvll.Module, func: omvll.Function):
        return True  

![Image](https://github.com/user-attachments/assets/2c70de3b-362d-4223-9ee6-a2b9c8c57594)

When I use this obfuscation in an iOS project, this error occurs. How do I deal with this?
